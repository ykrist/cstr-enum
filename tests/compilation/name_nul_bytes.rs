use cstr_enum::*;

#[derive(FromCStr, AsCStr)]
enum Enum{
  #[cstr(name="app\0le")]
  A,
}

fn main() {

}