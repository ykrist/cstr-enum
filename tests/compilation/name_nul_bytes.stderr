error: string cannot contain nul bytes
 --> $DIR/name_nul_bytes.rs:5:15
  |
5 |   #[cstr(name="app\0le")]
  |               ^^^^^^^^^
